packs/data/equipment-effects.db/effect-nosoi-charm-diplomacy.json

```
@@ -19,9 +19,9 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Nosoi Charm (Make A request)",
+                "label": "PF2E.SpecificRule.NosoiCharm.Request",
```

```suggestion
                "label": "PF2E.SpecificRule.Equipment.NosoiCharm.RequestLabel",
```
Let's move this to an equipment subsection in there. Ideally I would like to have started with more organization in the first place but now's a good time to start too. If you rebase this branch you'll see an equipment section was made recently. I'm also not convinced that these need labels but if they are localized it's not going to bother me much.